### PR TITLE
Name "Nether Star Chisel" only "Bedrockium Chisel" if in GTNH

### PR DESCRIPTION
**Pull request by glowredman**
GTNH Core Mod is present -> "Bedrockium Chisel"
Otherwise -> "Nether Star Chisel"

Localizations still work

### CODE REVIEW

File: src/main/java/team/chisel/Chisel.java [hunk 1]
```diff
@@ -138,6 +139,9 @@ public void preInit(FMLPreInitializationEvent event) {
 		if (Loader.isModLoaded("ForgeMultipart")) {
 			new FMPCompat().init();
 		}
+		if (Loader.isModLoaded("dreamcraft")) {
+		    gtnhLoaded = true;
```
Comment (miozune) [inline src/main/java/team/chisel/Chisel.java:143]: We need to detect GTNH before `Features.preInit()`, otherwise unlocalized name remains non-GTNH version

Comment (glowredman) [reply src/main/java/team/chisel/Chisel.java:143]: Can you elaborate that a bit? I don't know what you mean...

Comment (miozune) [reply src/main/java/team/chisel/Chisel.java:143]: Unlocalized names are set in `ItemChisel` constructor. They are instantiated in `Features.CHISEL#addItems`, which is called by `Features#loadItems`. As a result, unlocalized names are set before we set `gtnhLoaded = true`.

Comment (glowredman) [reply src/main/java/team/chisel/Chisel.java:143]: Ah, good catch

**Review by miozune — APPROVED** [review]

**Pull request merged**